### PR TITLE
DOC Update branch protection rules

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/.github/issues/198

### CODE REVIEW

File: en/12_Project_Governance/09_Repository_Management.md [hunk 1]
```diff
@@ -39,10 +39,24 @@ Some actions are restricted to avoid bad actors or accidental mistakes from caus
 
 If any exceptions to these rules need to be made for any reason, each exception must be approved in writing by the Silverstripe CMS Product Owner and only apply for a specified amount of time.
 
-#### Deleting branches
+#### Branch protection rules
+
+##### Updating branch and tag protection rulesets
+
+Use the `rulesets` command in [silverstripe/module-standardiser](https://github.com/silverstripe/module-standardiser) add and update the branch and tag protection rulesets for all support modules via the GitHub API.
```
Comment (ChloeSartorelli) [inline en/12_Project_Governance/09_Repository_Management.md]:
```suggestion
Use the `rulesets` command in [silverstripe/module-standardiser](https://github.com/silverstripe/module-standardiser) to add and update the branch and tag protection rulesets for all support modules via the GitHub API.
```

File: en/12_Project_Governance/09_Repository_Management.md [hunk 2]
```diff
@@ -39,10 +39,24 @@ Some actions are restricted to avoid bad actors or accidental mistakes from caus
...
+
+These rulesets restrict non-admins from various activities such as merging pull-requests without a review, creating branches, and pushing tags.
+
+##### Updating branch protection rule
 
 A branch protection rule to protect against deleting branches is added by going to `https://github.com/silverstripe/<repository>/settings/branches`, clicking "Add rule", inputting `[0-9]*` as the branch name pattern, and clicking create (without checking any of the checkboxes).
 
+There is not a REST API endpoint available to add this rule as it uses a pattern, so it must be done manually.
```
Comment (ChloeSartorelli) [inline en/12_Project_Governance/09_Repository_Management.md]:
```suggestion
There is no REST API endpoint available to add this rule with a pattern, so it must be done manually.
```

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: en/12_Project_Governance/09_Repository_Management.md (hunk 1 above)
Comment (emteknetnz) [reply en/12_Project_Governance/09_Repository_Management.md]: Updated

File: en/12_Project_Governance/09_Repository_Management.md (hunk 2 above)
Comment (emteknetnz) [reply en/12_Project_Governance/09_Repository_Management.md]: Updated

**Review by GuySartorelli — APPROVED** [review]
LGTM

**Pull request merged**